includes/base/element-base.php

```diff
@@ -450,7 +450,7 @@ public function add_link_attributes( $element, array $url_control, $overwrite =
 		$attributes = [];
 
 		if ( ! empty( $url_control['url'] ) ) {
-			$attributes['href'] = str_replace( 'javascript:', '', $url_control['url'] );
+			$attributes['href'] = (string) wp_kses_bad_protocol( $url_control['url'], [ 'http', 'https' ] );
```

Comment:
The result is already a string. Why do you convert it?

Reply:
Fixed.